content/contributing/opencue/getting-started-windows.md

```diff
@@ -0,0 +1,186 @@
+cuebot.facility:
+  local:
+  - localhost:8443
+```
+
+## Running CueSubmit
+
+CueSubmit lets you submit jobs to OpenCue, it is written in Python.
+
+To get setup and run it with PyCharm:
+
+1. Open PyCharm and choose “Open”, select the `cuesubmit` folder in the git repository.
+1. For the project interpreter, you can either set up a Venv (usually recommended if
+   you are working on other Python projects), or leave it as the default Python interpreter.
+1. PyCharm should prompt you to install ‘Package requirements’. If not, open the `setup.py`
+   and the banner should appear. Select “Install Requirements” and wait for the packages
+   to be installed. (PySide2 can take some time to install.)
+1. You will need to add additional content roots to find the `opencue` and `outline` libraries.
+   Under “File→Settings” find “Project: cuesubmit/Project Structure”; click the ➕ next
+   to “Add Content Root” and add the `pycue` folder. Do the same with the `pyoutline` folder.
+   Select “OK” to exit.
+1. Right click on `cuesubmit/ui/__main__.py` and select “Run”.
+
+## Running CueGUI
+
+CueGUI lets you monitor the status of jobs and rendering hosts. It is written in Python.
+
+To get setup and run it with PyCharm:
+
+1. Open PyCharm and choose “Open”, select the `cuegui` folder in the git repository.
+1. For the project interpreter, you can either set up a Venv (usually recommended if
+   you are working on other Python projects), or leave it as the default Python interpreter.
+1. PyCharm should prompt you to install ‘Package requirements’. If not, open the `setup.py`
+   and the banner should appear. Select “Install Requirements” and wait for the packages
+   to be installed. (PySide2 can take some time to install.)
+1. You will need to add additional content roots to find the `opencue` library.
+   Under “File→Settings” find “Project: cuegui/Project Structure”; click the ➕ next
+   to “Add Content Root” and add the `pycue` folder.
+   Select “OK” to exit.
+1. Right click on `cuegui/__main__.py` and select “Run”.
+
+## Running RQD
+
+RQD is the OpenCue rendering host agent. It is written in Python.
+
+To get set up and run it with PyCharm:
+
+1. Open PyCharm and choose “Open”, select the `rqd` folder in the git repository.
+1. For the project interpreter, you can either set up a Venv (usually recommended if
+   you are working on other Python projects), or leave it as the default Python interpreter.
+1. PyCharm should prompt you to install ‘Package requirements’. If not, open the `setup.py`
+   and the banner should appear. Select “Install Requirements” and wait for the packages
+   to be installed. 
+1. Right click on `rqd/__main__.py` and select “Run”.
+
+## Verify end-to-end
+
+Once you are running CueBot, CueGUI, and RQD simultaneously, you should be able to see
+the RQD host in CueGUI (in the “Monitor Hosts” section, check the “Auto-refresh” box):
+
+![A screenshot of CueGUI showing host](/docs/images/windows/verify_host.png)
```

Comment:
Could you crop this to focus on the most relevant UI elements for the reader? The ratio of the height:width should be much closer like 1:3. It's quite hard to see the details.

Reply:
Cropped